cmd/backend_kv_server.cpp

```diff
@@ -0,0 +1,95 @@
+/*
+   Copyright 2022 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#include <iostream>
+#include <string>
+#include <thread>
+
+#include <boost/asio/io_context.hpp>
+#include <boost/asio/signal_set.hpp>
+#include <boost/process/environment.hpp>
+#include <CLI/CLI.hpp>
+#include <magic_enum.hpp>
+
+#include <silkworm/chain/config.hpp>
+#include <silkworm/common/log.hpp>
+#include <silkworm/rpc/backend_server.hpp>
+
+int main(int argc, char* argv[]) {
+    const auto pid = boost::this_process::get_id();
+    const auto tid = std::this_thread::get_id();
+
+    CLI::App app{"ETHBACKEND & KV servers"};
+
+    std::string chain_id{"goerli"};
```

Comment:
Görli is a poor default because we don't support Clique in Silkworm yet. Why not mainnet?

Reply:
Yes, sure. It's just because on my machine I will test BE&KV server on Goerli for space reasons.